src/include/duckdb/common/allocator.hpp

```diff
@@ -48,9 +48,12 @@ class AllocatedData {
 
 class Allocator {
 public:
-	Allocator();
-	Allocator(allocate_function_ptr_t allocate_function_p, free_function_ptr_t free_function_p,
-	          reallocate_function_ptr_t reallocate_function_p, unique_ptr<PrivateAllocatorData> private_data);
+	DUCKDB_API Allocator();
+	DUCKDB_API Allocator(allocate_function_ptr_t allocate_function_p, free_function_ptr_t free_function_p,
+	                     reallocate_function_ptr_t reallocate_function_p,
+	                     unique_ptr<PrivateAllocatorData> private_data);
+
+	DUCKDB_API Allocator &operator=(Allocator &&allocator) noexcept;
```

Comment:
Can we use `DUCKDB_API Allocator &operator=(Allocator &&allocator) noexcept = default;` here?